### ISSUE TITLE
User email change or confirm - weird behaviour

### ISSUE THREAD

**Issue by idan-h** [OrchardCMS/OrchardCore#13929]
### To Reproduce
The whole user editing experience is a bit weird with everything that is related to email confirmation -
for example, as an administrator, you can't edit your own 'email confirmed' for some reason.
if changing emails is permitted, then the user can validate his email and then change it, and 'email confirmed' will still be true, although its not validated.

I think the real solution is to ditch the CRUD like interface with everything related to the email change, and use specific endpoint for it, and change the email only after it is confirmed.

**Comment by MikeAlhayek** [comment]
yes email confirmation should have a confirmation process associated with it.

**Comment by MikeAlhayek** [comment]
@idan-h Feel free to submit a PR that would remove `EmailConfirmed` from the `UserDisplayDriver`. However, it may be worth adding a green tag next to the email saying `confirmed` or black `unconfirmed`.

We already have 2 ways to validate email address. 
1. `OrchardCore.Users.Registration`
2. `Two-Factor Email Method`

**Comment by idan-h** [comment]
I am not big on the front side, I will see if bootstrap has some tags and will try to use that
Anyway its good idea to merge my old pull request first, so there wouldn't be conflicts

**Comment by MikeAlhayek** [comment]
you can use this `<span class="badge ta-badge">@T["Unverified"]</span>` and `<span class="badge bg-sucess">@T["Verified"]</span>`

**Comment by gvkries** [comment]
This has been changed in the main branch.
